readthedocs/builds/views.py

```diff
@@ -82,6 +84,30 @@ class BuildDetail(BuildBase, DetailView):
     def get_context_data(self, **kwargs):
         context = super(BuildDetail, self).get_context_data(**kwargs)
         context['project'] = self.project
+        scheme = ("https://github.com/rtfd/readthedocs.org/issues/new"
+                  "?title={title}{build_id}"
+                  "&body={body}")
+
+        body = ("# Details:\n\n"
+                "*Project URL: https://readthedocs.org/projects/{projname}/\n"
+                "*Build URL(if applicable): https://readthedocs.org{build_path}\n"
+                "*Read the Docs username(if applicable): {uname}\n\n"
+                "## Expected Result\n\n"
+                "*A description of what you wanted to happen*\n\n"
+                "## Actual Result\n\n"
+                "*A description of what actually happened*")
+
+        body_dict = {'projname': self.project,
```

Comment:
I thinks is more clear if you use this on the format method directly, since isn't used on other parts of the code.

Reply:
Done!